### ISSUE TITLE
Issue with VSCode completion of filepaths containing $lib

### ISSUE THREAD

**Issue by vlrevolution** [sveltejs/kit#9748]
### Describe the bug

Using the skeleton project, currently file path completions containing $lib for some reason do not work, this happens (it puts \ instead of $lib):
![2023-04-22 13_38_19-Window](https://user-images.githubusercontent.com/67480746/233779968-cee4aae8-af48-485d-8171-39df24c6a94b.png)
![2023-04-22 13_46_27-Window](https://user-images.githubusercontent.com/67480746/233779972-4c6afdde-26a2-487d-9dc6-512c52bb764a.png)

Anybody else experiencing this?

### Reproduction

Try to auto complete filepaths when doing import containing $lib

### System Info

```Shell
System:
    OS: Linux 5.4 Debian GNU/Linux 11 (bullseye) 11 (bullseye)
    CPU: (32) x64 AMD Ryzen 9 5950X 16-Core Processor
    Memory: 22.08 GB / 24.99 GB
    Container: Yes
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 18.13.0 - ~/.nvm/versions/node/v18.13.0/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 9.4.0 - ~/.nvm/versions/node/v18.13.0/bin/npm
  Browsers:
    Chrome: 108.0.5359.124
    Chromium: 106.0.5249.119
  npmPackages:
    @sveltejs/adapter-auto: ^2.0.0 => 2.0.0
    @sveltejs/kit: ^1.5.0 => 1.15.7
    svelte: ^3.54.0 => 3.58.0
    vite: ^4.2.0 => 4.3.1
```

### Severity

serious, but I can work around it

**Comment by hughest64** [comment]
I experience this issue with `$lib` and `$app`  imports.

![Screenshot_237](https://user-images.githubusercontent.com/18269570/233791270-83570067-f60f-4a63-93e1-32cafba2d317.png)
![Screenshot_238](https://user-images.githubusercontent.com/18269570/233791308-1e042a2f-c69f-47dc-9261-7ca893f6b0b6.png)

**Comment by jasonlyu123** [comment]
Please do not cross-post like this. This is an upstream TypeScript issue that will be fixed in Typescript 5.1.

**Comment by vlrevolution** [comment]
Sorry @jasonlyu123, was not sure which package was responsible!

**Comment by jasonlyu123** [comment]
It is fixed in TypeScript 5.1, bundled in VSCode to 1.79 and Svelte For VSCode 107.6.0.
